beeflow/client/bee_client.py

```diff
@@ -173,6 +197,9 @@ def submit(wf_name: str = typer.Argument(..., help='The workflow name'),
         error_exit('Could not reach WF Manager.')
 
     if resp.status_code != requests.codes.created:  # pylint: disable=no-member
+        if resp.status_code == 400:
```

Comment:
I think I get the logic, but why did you pick 400 in particular here?

Reply:
I think 400 means something like the user submitted bad arguments or a bad request (not our fault). So this would be returned if the user mistyped something or sent a bad workflow file.

Reply:
Sounds reasonable.